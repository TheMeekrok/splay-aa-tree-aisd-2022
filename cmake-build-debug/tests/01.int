insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
insert 940
