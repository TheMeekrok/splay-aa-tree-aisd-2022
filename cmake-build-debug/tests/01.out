Splay Tree execution time (ms): 75.806
